.github/workflows/conformance.yml

```diff
@@ -0,0 +1,34 @@
+name: Ruby Conformance CI
+on: push
```

Comment:
Should we use the same triggers as the unit and lint tests? (i.e. run on pull requests and merges to main). I'd also like to include workflow_dispatch so admins can trigger manually.

```
on:
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
  workflow_dispatch:
```

Reply:
Done